samza-core/src/main/java/org/apache/samza/table/remote/couchbase/CouchbaseTableFunctionBase.java

```diff
@@ -25,9 +25,11 @@
 import com.google.common.collect.ImmutableList;
 import java.util.Collection;
 import java.util.List;
+import java.util.concurrent.TimeUnit;
 import org.apache.samza.context.Context;
 import org.apache.samza.operators.functions.ClosableFunction;
 import org.apache.samza.operators.functions.InitableFunction;
+import org.apache.samza.serializers.Serde;
 
 
 public class CouchbaseTableFunctionBase<V> implements InitableFunction, ClosableFunction {
```

Comment:
Can we use name Base..., please be mindful about convention.

Reply:
I see. I will rename it.